src/BugsnagUnity/Native/Cocoa/NativeClient.cs

```diff
@@ -449,5 +449,9 @@ public bool ShouldAttemptDelivery()
         {
             return true;
         }
+
+        public void AddOnSession()
+        {
```

Comment:
I think for completeness we should implement the Cocoa side too as this has the equivalent `bugsnag_registerForSessionCallbacks` method.